### DISCUSSION TITLE
Long load time on mobile version

### DISCUSSION THREAD

**Discussion by pclahoud** [xournalpp/xournalpp#4557] [Q&A]
I am trying to open an xopp file that I made from a pdf on my laptop that is a couple hundred pages long. It is taking a long time to load on my tablet. Is there anything I can do?

**Comment by rolandlo** [comment]
That seems to be an issue of the mobile version. Please report it [there](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile). Although I must warn that the question has been raised, if the developer of the mobile version is still active, see https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/issues/56.

Just for clarity: What tablet/what OS do you have?

**Reply by pclahoud** [reply]
I am using the Galaxy Tab S8+ running Android 13 under One UI 5
